en/docs/get-started/quick-start-guide.md

```diff
@@ -29,7 +31,7 @@ Choreo maintains two environments by default: development and production. Let's
 
 1. Click **Deploy** in the left pane. 
 
-2. Go to the **Build Area** card and click **Deploy**.
+2. Go to the **Build Area** card and click **Build and Deploy**.
```

Comment:
**Build and Deploy** should be replaced with **Config & Deploy** right?

Reply:
It is Build & Deploy because this implementation does not have any configurable variables.